snmp/datadog_checks/snmp/data/profiles/_cisco-generic.yaml

```diff
@@ -1,7 +1,6 @@
-# Base profile for Cisco devices
+# Metrics mixin for Cisco devices.
```

Comment:
How confident are we changing these mixin file names? Could people be using them for their own profiles in production?

Reply:
Converted existing files to shims around the renamed mixins.